packages/admin/cms-admin/src/dam/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -57,6 +57,7 @@ export const DamFileDownloadLinkBlock: BlockInterface<DamFileDownloadLinkBlockDa
                         name
                         fileUrl
                         size
+                        mimetype
```

Comment:
should it be `mimeType` camel case, like `fileUrl` ? Is there a naming convention?

Reply:
In the entity we use mimetype, so I also used it here 

https://github.com/vivid-planet/comet/blob/f911089c517de1063817cd07e82346cec19a4b94/packages/api/cms-api/src/dam/files/entities/file.entity.ts#L75